stubs/protobuf/@tests/stubtest_allowlist.txt

```diff
@@ -3,6 +3,10 @@
 google.protobuf\..*_pb2\..*
 
 google.protobuf.internal.containers.BaseContainer.sort
+# While Message and Descriptor are both defined with a null DESCRIPTOR,
+# sublcasses of Message and instances of EnumTypeWrapper require this value to
+# be set, and since These type stubs are intended for use with protoc generated
```

Comment:
```suggestion
# subclasses of Message and instances of EnumTypeWrapper require this value to
# be set, and since these type stubs are intended for use with protoc-generated
```

Reply:
good catch, thanks!